### ISSUE TITLE
[REVIEW]: profileR: An R package for profile analysis

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#1941]
**Submitting author:** @cddesja (<a href="http://orcid.org/0000-0002-8234-9665">Christopher Desjardins</a>)
**Repository:** <a href="https://github.com/cddesja/profileR" target ="_blank">https://github.com/cddesja/profileR</a>
**Version:** v0.3-6
**Editor:** @marcosvital
**Reviewers:** @wjakethompson, @jrosen48
**Archive:** <a href="https://doi.org/10.5281/zenodo.3711046" target="_blank">10.5281/zenodo.3711046</a>

## Status

[![status](https://joss.theoj.org/papers/85980d724b2b7fc7d10f7fca27fbce92/status.svg)](https://joss.theoj.org/papers/85980d724b2b7fc7d10f7fca27fbce92)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/85980d724b2b7fc7d10f7fca27fbce92"><img src="https://joss.theoj.org/papers/85980d724b2b7fc7d10f7fca27fbce92/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/85980d724b2b7fc7d10f7fca27fbce92/status.svg)](https://joss.theoj.org/papers/85980d724b2b7fc7d10f7fca27fbce92)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/cddesja/profileR" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@wjakethompson & @jrosen48 & @markhwhiteii , please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @marcosvital know.

✨ **Please try and complete your review in the next two weeks** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/cddesja/profileR">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@cddesja) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/cddesja/profileR">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@cddesja) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [ ] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/cddesja/profileR">repository url</a>?
- [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [ ] **Contribution and authorship:** Has the submitting author (@cddesja) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [ ] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [ ] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [ ] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [ ] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [ ] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @wjakethompson, @jrosen48, @markhwhiteii  it looks like you're currently assigned to review this paper :tada:.

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.1037//1082-989X.7.4.468 is OK

MISSING DOIs

- https://doi.org/10.1080/00273170903187665 may be missing for title: Factor Analytic Modeling of Within Person Variation in Score Profiles
- https://doi.org/10.1201/b20498 may be missing for title: Handbook of educational measurement and psychometrics using R

INVALID DOIs

- None
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01941/joss.01941/10.21105.joss.01941.pdf)

**Comment by jrosen48** [comment]
Hi, sorry for my delay on this. I will aim to proceed with my review this week.

**Comment by cddesja** [comment]
Is there an update on the timeline for the review?

**Comment by marcosvital** [comment]
Hi, everyone!

@cddesja , sorry about the delayed reply, I'm still organizing myself after a few weeks out of office.

@wjakethompson , @jrosen48 and @markhwhiteii, let us know when you have the time to look into this review, ok? And please feel free to contact me if you have any questions about the reviewing process. 

Best regards

Marcos

**Comment by jrosen48** [comment]
Hi @marcosvital, I seem to be unable to check the checkbox items - can you advise if there is something I can do? thanks!

**Comment by kthyng** [comment]
Hi @jrosen48 — did you receive an invitation to edit by email? You need to accept that to gain proper permissions on this issue. Let me know if you don't have the invitation.

**Comment by jrosen48** [comment]
Hi, I checked my email, but didn't see this! Is there any chance that you would be able to invite me again? Thank you!

**Comment by kthyng** [comment]
@jrosen48 hm, check out the list up above under "Reviewer instructions & questions" and see if you have done both those things (being logged into github and it gives a page for invites).

**Comment by jrosen48** [comment]
doh, so sorry - thanks - that did it. I'm sorry!

**Comment by wjakethompson** [comment]
I have completed my initial review. Overall, it looks like great package! A summary of why some boxes have not been checked:

### Acceptance blockers
* **Example usage**: See cddesja/profileR#28.
* **Automated tests**: See cddesja/profileR#29.
* **Community guidelines**: See cddesja/profileR#30.


### Other issues
* **License**: A license is included, conflicts with the license listed in the DESCRIPTION. See cddesja/profileR#32.
* **Installation instructions**: Instructions are included, and work. But I think CRAN instructions should also be included. See cddesja/profileR#27.
* **State of the field**: No comparisons are made to other packages. See cddesja/profileR#31.
* **References**: Assuming packages are added for state of the field, appropriate references would also need to be included.

**Comment by cddesja** [comment]
@jrosen48 and @wjakethompson, I believe we have addressed all the outstanding issues. @jrosen48, please let us know if you think the state of the field has been described adequately.

**Comment by cddesja** [comment]
@wjakethompson, we are wondering if there are changes are adequate for you and whether you have any additional changes?

@kthyng, we haven't heard back from @markhwhiteii. should we try and get another reviewer or if both @wjakethompson and @jrosen48 are satisfied with the state of the paper, is that sufficient?

**Comment by wjakethompson** [comment]
@cddesja Everything looks good to me!

**Comment by kthyng** [comment]
@cddesja Let's check with @marcosvital, who is the handling editor.

@marcosvital Did you want to have 3 reviewers for this submission in particular or are you ok with two?

**Comment by marcosvital** [comment]
Hi, @kthyng , yes, I'm comfortable with moving on with two reviews, as they are both satisfied with the paper.

@wjakethompson ad @jrosen48 , thank you both so much for taking time to review and make suggestions.
@jrosen48, I noticed that there is one last box left unchecked in you review, regarding "Functionality documentation". Do you fell there is still something concerning this that the authors should address?

@cddesja , we are almost done here. Once we confirm that everything is ok to move forward, you will need to archive the last release of the package (on Zenodo, figshare, or other), if you already didn't do this, post the version number and archive DOI here in this issue.

**Comment by cddesja** [comment]
@marcosvital, sounds good. Are you waiting on me, right now? Or are we waiting on 

> @jrosen48, I noticed that there is one last box left unchecked in you review, regarding "Functionality documentation". Do you fell there is still something concerning this that the authors should address?

**Comment by marcosvital** [comment]
Hi, @cddesja! I was waiting to know about the unchecked box. But since it's just a single item from the whole list, I'll make a final and quick check on everything myself, and we'll move on, ok? Will be back here soon.

**Comment by marcosvital** [comment]
Hello again, @cddesja, I think that everything seems ok, so let's proceed with our last steps towards acceptance!

**Comment by marcosvital** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01941/joss.01941/10.21105.joss.01941.pdf)

**Comment by marcosvital** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.1037//1082-989X.7.4.468 is OK

MISSING DOIs

- https://doi.org/10.1080/00273170903187665 may be missing for title: Factor Analytic Modeling of Within Person Variation in Score Profiles
- https://doi.org/10.1201/b20498 may be missing for title: Handbook of educational measurement and psychometrics using R
- https://doi.org/10.21105/joss.00978 may be missing for title: tidyLPA: An R package to easily carry out Latent Profile Analysis (LPA) using open-source or commercial software
- https://doi.org/10.32614/rj-2016-021 may be missing for title: mclust 5: clustering, classification and density estimation using Gaussian finite mixture models

INVALID DOIs

- None
```

**Comment by marcosvital** [comment]
@cddesja, please check out the manuscript references: they should have the DOIs listed, ok?

**Comment by marcosvital** [comment]
@markhwhiteii, as JOSS requires only two reviewers, we are proceeding with the paper acceptance. Since the reviewers are listed on the published paper, I'll remove your name from the list. I understand that taking time to review a paper is not easy, so thanks for trying to help with this one, and I hope we can count on you in future reviewing tasks.

**Comment by marcosvital** [comment]
@whedon remove @markhwhiteii  as reviewer

**Comment by whedon** [comment]
OK, @markhwhiteii  is no longer a reviewer

**Comment by okanbulut** [comment]
> @cddesja, please check out the manuscript references: they should have the DOIs listed, ok?

@marcosvital, I fixed the missing DOIs in the paper. They all should be good now. Thank for catching that.

**Comment by cddesja** [comment]
@marcosvital, I'd like to make a reference to our two vignettes included in the package by adding the following sentence bolded below. Is this okay? I'll add the citation to the paper.bib if this is fine.

The R package ``profileR``, currently in version 0.3-6 on the comprehensive R archive network (CRAN), implements profile reliability [@bulut2013between; @bulut2017], criterion-related profile analysis [@davison_identifying_2002], profile analysis via multidimensional scaling [@pams], moderated profile analysis, profile analysis by group, and a within-person factor model to derive score profiles [@davison2009factor] as well as a variety of graphical methods to visualize profiles. Related profile analysis packages exist in R, such as ``tidyLPA`` [@rosenberg2019tidylpa] and ``mclust`` [@mclust], as well as in Mplus [@muthen2004mplus], however, our approach does not rely on mixture modeling and aims to provide access to a wider range of profile analytic techniques. **An extensive vignette documenting ``profileR`` [@bulut2020] is included with the package as well as vignette to ensure the functionality of the package locally.** 

If this is fine with you, then I'll add this to the paper.md and bib file then I'll archive the package on Zendodo.

**Comment by arfon** [comment]
Dear authors and reviewers

We wanted to notify you that in light of the current COVID-19 pandemic, JOSS has decided to suspend submission of new manuscripts and to handle existing manuscripts (such as this one) on a "best efforts basis". We understand that you may need to attend to more pressing issues than completing a review or updating a repository in response to a review. If this is the case, a quick note indicating that you need to put a "pause" on your involvement with a review would be appreciated but is not required.

Thanks in advance for your understanding.

_Arfon Smith, Editor in Chief, on behalf of the JOSS editorial team._

**Comment by cddesja** [comment]
@arfon, the paper is done. I posted the software on Zendodo.
DOI: 10.5281/zenodo.3711046
Version: 0.3-6
The paper.md and paper.bib files are done.

**Comment by arfon** [comment]
👍 roger that @cddesja. We'll just wait to hear back from @marcosvital before finalizing the accept/publish steps.

**Comment by marcosvital** [comment]
Everything looks fine, so we can move on to acceptance! Let's make a final check on references and proof.

**Comment by marcosvital** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.1037//1082-989X.7.4.468 is OK
- 10.1080/00273171.2016.1253452 is OK
- 10.1080/00273170903187665 is OK
- 10.1201/b20498 is OK
- 10.21105/joss.00978 is OK
- 10.32614/rj-2016-021 is OK

MISSING DOIs

- None

INVALID DOIs

- https://doi.org/10.31234/osf.io/sgy8m is INVALID because of 'https://doi.org/' prefix
```

**Comment by marcosvital** [comment]
@cddesja , there is a final invalid DOI. Can you check and correct it?

**Comment by okanbulut** [comment]
@cddesja @marcosvital , the invalid DOI has been fixed now. Thanks!

**Comment by cddesja** [comment]
Can I look at the final proof just to make it looks okay? Thanks for fixing the DOI, Okan. 
Chris

**Comment by marcosvital** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.1037//1082-989X.7.4.468 is OK
- 10.1080/00273171.2016.1253452 is OK
- 10.1080/00273170903187665 is OK
- 10.1201/b20498 is OK
- 10.21105/joss.00978 is OK
- 10.31234/osf.io/sgy8m is OK
- 10.32614/rj-2016-021 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by marcosvital** [comment]
Thank you for correcting it, @okanbulut , everything seems fine!

@cddesja , I'll generate another proof.

**Comment by marcosvital** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01941/joss.01941/10.21105.joss.01941.pdf)

**Comment by cddesja** [comment]
Okay, I changed profileR to ``profileR`` and uploaded the new paper.md.
Everything else looks good to me.

**Comment by marcosvital** [comment]
Good! I'll start the final steps towards acceptance.

**Comment by marcosvital** [comment]
@whedon set 10.21105.joss.01941 as archive

**Comment by whedon** [comment]
10.21105.joss.01941 doesn't look like an archive DOI.

**Comment by marcosvital** [comment]
Sorry, my mistake.

**Comment by marcosvital** [comment]
@whedon set 10.5281/zenodo.3711046 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.3711046" target="_blank">10.5281/zenodo.3711046</a> is the archive.

**Comment by marcosvital** [comment]
@whedon set v0.3-6 as version

**Comment by whedon** [comment]
OK. v0.3-6 is the version.

**Comment by marcosvital** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.1037//1082-989X.7.4.468 is OK
- 10.1080/00273171.2016.1253452 is OK
- 10.1080/00273170903187665 is OK
- 10.1201/b20498 is OK
- 10.21105/joss.00978 is OK
- 10.31234/osf.io/sgy8m is OK
- 10.32614/rj-2016-021 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published

. Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/1373

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/1373, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by arfon** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1240019838568931333) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/1374
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.01941](https://doi.org/10.21105/joss.01941)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? notify your editorial technical team...

**Comment by arfon** [comment]
@wjakethompson, @jrosen48 - many thanks for your reviews here and to @marcosvital for editing this submission ✨ 

@cddesja - your paper is now accepted into JOSS :zap::rocket::boom:

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.01941/status.svg)](https://doi.org/10.21105/joss.01941)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.01941">
  <img src="https://joss.theoj.org/papers/10.21105/joss.01941/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.01941/status.svg
   :target: https://doi.org/10.21105/joss.01941
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.01941/status.svg)](https://doi.org/10.21105/joss.01941)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss

**Comment by okanbulut** [comment]
> @wjakethompson, @jrosen48 - many thanks for your reviews here and to @marcosvital for editing this submission ✨
> 
> @cddesja - your paper is now accepted into JOSS ⚡️🚀💥

Thank you very much!
